src/ChannelAudience/Private.js

```diff
@@ -3,9 +3,10 @@
 const ChannelAudience = require('../ChannelAudience');
 
 /**
- * Audience class representing other players in the area as the sender
+ * Audience class representing a specific targeted player.
+ * Example: `tell` command or `whisper` command.
  */
-class Area extends ChannelAudience {
+class PrivateAudience extends ChannelAudience {
```

Comment:
I'll change it, I just noticed the copy/paste error and thought I'd make the names a bit clearer.